.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,3 +16,21 @@ Addresses issue #
 - [ ] My code has proper inline documentation.
 - [ ] I have added a QA Brief on the issue linked above.
 - [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).
+
+---------------
+
+## Code Reviewer Checklist
+
+- [ ] Run the code.
+- [ ] Ensure the acceptance criteria are covered.
+- [ ] Stay open-minded.
```

Comment:
I think this reads a bit weird now as its own requirement - you should always stay open minded. :) This point was more specifically about following the IB where appropriate, but also allowing for diversion from the IB if it makes sense.

Maybe something like "Check whether the approach from the IB still makes sense here in practice"? @aaemnnosttv Any wording suggestion?

Reply:
The points in the handbook have changed, so let's update this accordingly.
```suggestion
- [ ] Reassess the implementation with the IB.
```